tests/libc/CMakeLists.txt

```diff
@@ -17,7 +18,19 @@ if (UNIX)
 	add_subdirectory(enc)
 endif()
 
+string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UPPER)
 foreach(testcase ${alltests})
+    if (CMAKE_C_COMPILER_ID MATCHES Clang)
+	if (BUILD_TYPE_UPPER MATCHES DEBUG)
+	    if (${testcase} IN_LIST clang_debug_skipped_libc_tests)
+		continue ()
+		endif ()
```

Comment:
Just a petting formatting thing, the `endif()` here and below are too indented, otherwise looks great, thanks 😄